### PR TITLE
Update Readme.md - how to start setion at the end

### PR DESCRIPTION
**Pull request by mmnosek**
Solves #53

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,3 +334,13 @@ public partial class InvoicePage : Json, IBound<Invoice>
 ```
 
 Method marked with this attribute should be public and static, return `Permission` and accept context as its argument. Whatever permission it returns is then checked to see if the user can access the page.
+
+# How to start
+Minimum reasonable configuration to start playing around with this library is to use sample [apps provided by Starcounter](https://github.com/StarcounterApps):
```
Comment (joozek78) [inline README.md]: It's not a minimal configuration, it's a quick start configuration. There's a difference. Minimal would be just SignIn or equivalent

Comment (joozek78) [inline README.md]: It's not a minimal configuration, it's a quick start configuration. There's a difference. Minimal would be just SignIn or equivalent

**Comment by mmnosek** [comment]
Changed

**Comment by joozek78** [comment]
IMO quickstart should not be at the end ;p
Maybe also add a table of contents?

**Comment by mmnosek** [comment]
Check once again please :)

**Pull request merged**